docs/code-reviews/inclusion-in-code-review.md

```diff
@@ -0,0 +1,59 @@
+# Inclusion in Code Review
+
+You may be thinking why care about inclusiveness in code reviews. Below are some points which emphasize why inclusivity in code reviews is important:

```

Comment:
We could also be direct here e.g., "Inclusiveness in code review is important because:"

Reply:
done